[2024-03-01T16:35:14.328+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T10:52:00+00:00 [queued]>
[2024-03-01T16:35:14.334+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T10:52:00+00:00 [queued]>
[2024-03-01T16:35:14.334+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-01T16:35:14.357+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): zipallfiles_and_placeatrequiredlocation> on 2024-03-01 10:52:00+00:00
[2024-03-01T16:35:14.363+0530] {standard_task_runner.py:60} INFO - Started process 22926 to run task
[2024-03-01T16:35:14.367+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'datapipeline_Take4', 'zipallfiles_and_placeatrequiredlocation', 'scheduled__2024-03-01T10:52:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/datapipeline_best(2).py', '--cfg-path', '/tmp/tmpbf01n0hy']
[2024-03-01T16:35:14.369+0530] {standard_task_runner.py:88} INFO - Job 125: Subtask zipallfiles_and_placeatrequiredlocation
[2024-03-01T16:35:14.422+0530] {task_command.py:423} INFO - Running <TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T10:52:00+00:00 [running]> on host arun-VM
[2024-03-01T16:35:14.490+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='datapipeline_Take4' AIRFLOW_CTX_TASK_ID='zipallfiles_and_placeatrequiredlocation' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T10:52:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T10:52:00+00:00'
[2024-03-01T16:35:14.521+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-03-01T16:35:14.526+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=datapipeline_Take4, task_id=zipallfiles_and_placeatrequiredlocation, execution_date=20240301T105200, start_date=20240301T110514, end_date=20240301T110514
[2024-03-01T16:35:14.588+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T16:35:14.603+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
